tests/test_dipole.py

```diff
@@ -23,6 +23,26 @@ def test_dipole_off():
     assert not torch.allclose(outbeam_dipole_on.sigma_x, outbeam_drift.sigma_x)
 
 
+def test_dipole_focussing():
+    """
+    Test that a dipole with focussing moment behaves like a quadrupole.
+    """
+    dipole = Dipole(length=torch.tensor([1.0]), k1=torch.tensor([10.0]))
+    quad = Quadrupole(length=torch.tensor([1.0]), k1=torch.tensor([10.0]))
+    incoming_beam = ParameterBeam.from_parameters(
+        sigma_px=torch.tensor([2e-7]), sigma_py=torch.tensor([2e-7])
+    )
+    outbeam_dipole_on = dipole(incoming_beam)
+    outbeam_quad = quad(incoming_beam)
```

Comment:
```suggestion
    outbeam_dipole_on = dipole.track(incoming_beam)
    outbeam_quadrupole = quadrupole.track(incoming_beam)
```

Reply:
I had no idea that this worked. Totally on accident

Reply:
😄 ... this is actually how it was done in old versions of Cheetah. But I would argue the `track` method is now preferred, simply because it's more explicit.